### ISSUE TITLE
[Feature Request] Inline PDF Preview

### ISSUE THREAD

**Issue by markphilpot** [HappenApps/Quiver#406]
One thing I miss from Evernote is when I save a PDF into a note, there is an inline preview/viewer of the PDF directly in the note (rather than clicking the underlying link to open another app)

Use Case Example (in combo with a web clipper):
- In browser find a pdf manual of appliance (for example)
- Clip PDF to a note
- On viewing the note, the PDF is rendered directly (rather than clicking a link to open Preview.app)

**Comment by ylian** [comment]
Yes, I'm aware of that. There was a little technical difficulty, but I will figure it out.

**Comment by paulelms** [comment]
+1 May be invent a separate cell «file» with built-in quick look?

**Comment by mgwio** [comment]
I'd like to third this. Literally the only thing I miss from Evernote that's keeping me from dropping that ecosystem entirely. The convenience of 'quick-look'ing at PDF previews in notes is invaluable.

**Comment by fharper** [comment]
+1
